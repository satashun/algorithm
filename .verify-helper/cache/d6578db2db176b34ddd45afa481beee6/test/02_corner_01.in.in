100000 17
0 100000 1
1 1
1 50000
1 100000
0 1 1
1 100000
1 50000
1 1
0 49999 1
1 100000
1 50000
1 1
0 1 1
0 10000 1
1 100000
1 50000
1 1
