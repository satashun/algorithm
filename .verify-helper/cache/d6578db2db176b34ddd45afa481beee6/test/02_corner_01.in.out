100000 1
49999 0
99999 0
99999 0
49999 0
1 1
99999 0
49998 0
1 1
99999 0
49998 0
1 2
